index.html

```diff
@@ -7,42 +7,108 @@
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
-      text-align: center;
+      text-align: center;                 
+      font-size: 24px;
+      font-weight: bold;
+      color: #2f4f4f;
+            }
     }
   </style>
+    <style>
+    .mainparagraph {
+      text-align: center;                 
+      font-size: 20px;
+      color: black;
+            }
+    }
+  </style>
+    <style>
+    .auxparagraph {
+      text-align: right;                 
+      font-size: 10px;
+      font-style: oblique;
+      color: #333;
+            }
+    }
+  </style>
+   
+
+   <style>
+  table, th, td {
+   border: 1px solid grey;
+   height: 25px;
+   vertical-align: bottom;
+   text-align: left;
+   color: C0C0C0;
```

Comment:
Hex codes need to have a `#` before them. Realize that this might make the hover effect work in combination with the hover effect, although it has the opposite effect when a row is not hovered, making the text too light to see.